src/griddyn/simulation/gridSimulation.cpp

```diff
@@ -119,16 +119,16 @@ void gridSimulation::timestep (coreTime time, const IOdata &inputs, const solver
 void gridSimulation::saveRecorders ()
 {
     // save the recorder files
-    for (auto &col : collectorList)
+    for (auto const& col : collectorList)
```

Comment:
I am not sure this should be const  flush may not be a constant operation

Reply:
Good point. I'm somewhat interested in how this compiled...